css/src/components/hero.scss

```diff
@@ -34,6 +34,52 @@ $hero-image-width-widescreen: calc((100% - 2 * $layout-widescreen-gap) * 0.55) !
 		}
 	}
 
+	.hero-content-right {
+		display: flex;
+		flex-grow: 1;
+		padding-block: $hero-padding-xs;
+
+		@include tablet {
+			justify-content: flex-end;
+			padding-block: $hero-padding-sm;
+			padding-inline-start: $hero-padding-xs;
+		}
+
+		@include desktop {
+			width: $hero-content-fluid-width;
+			min-height: $hero-min-height-default;
+			padding-block: $hero-padding-default;
+			padding-inline-start: $hero-padding-sm;
+			padding-inline-end: $hero-padding-sm;
+		}
+
+		.hero-right-card {
+			align-self: flex-start;
+			background: #ffffffd9;
```

Comment:
after things have solidified a bit more here, let's consider how we can remove the static color values in favor of themed

Reply:
Ill adjust it now.